sections/main-password-header.liquid

```diff
@@ -1,15 +1,21 @@
+<style type="text/css">
+  .password-logo {
+    max-width: {{ section.settings.logo_max_width }}px;
+  }
+</style>
+
 <div class="color-{{ section.settings.color_scheme }} gradient">
   <div class="password-header">
     {%- if section.settings.logo -%}
```

Comment:
```suggestion
    {%- if section.settings.logo != blank -%}
```

Apologies for not noticing this in my previous review and it's totally a non-issue. I'm simply making this suggestion to make things consistent. Feel free to ignore this.